packages/dynamoose/lib/Table/utilities.ts

```diff
@@ -275,6 +290,47 @@ export async function updateTable (table: Table): Promise<void> {
 					object.StreamSpecification.StreamViewType = updateStreamViewType;
 				}
 
+				await ddb(instance, "updateTable", object);
+				await waitForActive(table)();
+			}
+		}
+	}
+	// Replication
+	if (updateAll || (table.getInternalProperties(internalProperties).options.update as TableUpdateOptions[]).includes(TableUpdateOptions.replication)) {
+		const tableDetails = (await getTableDetails(table)).Table;
+		const replicationOptions = table.getInternalProperties(internalProperties).options.replication;
+		
+		if (replicationOptions?.regions?.length > 0) {
+			// Get current replication regions
+			const currentReplicationRegions = tableDetails.ReplicationSpecification?.Regions || [];
+			const desiredReplicationRegions = replicationOptions.regions;
+			
+			// Check if replication settings need to be updated
+			const regionsNeedUpdate = !utils.array_equals(currentReplicationRegions.sort(), desiredReplicationRegions.sort());
+			
+			if (regionsNeedUpdate) {
+				// Ensure streams are enabled for replication
+				if (!tableDetails.StreamSpecification?.StreamEnabled) {
+					// Enable streams first if needed
```

Comment:
We don't want to enable streams automatically on the users behalf. Just throw an error if it's not a valid configuration.